src/care/LoopFuser.h

```diff
@@ -374,9 +374,15 @@ class FusedActionsObserver : public FusedActions {
       for (auto & priority_action : m_fused_action_order) {
          FusedActions * const & actions = priority_action.second;
          if (actions -> actionCount() > 0) {
-            actions->flushActions(async, fileName, lineNumber);
+            if (verbose) {
+               printf("flushing actions at priority %g\n", priority_action.first);
+            }
+            actions->flushActions(true, fileName, lineNumber);
```

Comment:
Shouldn't you be using `async` instead of `true`?

Reply:
@robinson96 was this change intentional or leftover from debugging?

Reply:
OK, that makes more sense.

Reply:
I would just add a comment to the code explaining this.